packages/agoric-cli/test/agops-perf-smoketest.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+// @ts-check
+
+// #region ambient authority
+import process from 'process';
+import { $ } from 'zx';
+
+const { env, argv: nodeArgv, exit } = process;
+const echo = (...args) => console.log(...args);
+// #endregion
+
+// nodeArgv = ['node', 'smoketest', 'wallet']
+// argv = ['smoketest', 'wallet']
+const argv = nodeArgv.slice(1);
+
+if (!env.AGORIC_NET) {
+  echo('AGORIC_NET env not set');
+  echo();
+  echo('e.g. AGORIC_NET=ollinet (or export to save typing it each time)');
+  echo();
+  echo(`To test locally, AGORIC_NET=local and have the following running:
+# freshen sdk
+yarn install && yarn build
+
+# local chain running with wallet provisioned
+packages/inter-protocol/scripts/start-local-chain.sh YOUR_ACCOUNT_KEY
+`);
+  exit(1);
+}
+
+const WALLET = argv[1];
+
+if (!WALLET) {
+  echo(`USAGE: ${argv[0]} key`);
+  echo('You can reference by name: agd keys list');
+  echo(
+    'Make sure it has been provisioned by the faucet: https://$AGORIC_NET.faucet.agoric.net/',
+  );
+  exit(1);
+}
+
+// NB: fee percentages must be at least the governed param values
+
+// NOTE: USDC_axl = ibc/usdt1234
+// perf test wantMinted
+let OFFER = await $`mktemp -t agops.XXX`;
+await $`bin/agops psm swap --wantMinted 0.01 --feePct 0.01 --pair IST.USDC_axl >|${OFFER}`;
```

Comment:
I'm gung ho about zx but there's a downside to JS syntax for for these smoketest scripts. I tend to run them interactively by copying lines over to my shell. This makes that harder to do.

For this script, I'm more inclined to keep the shell commands but place them into a [Markdown file that zx would run](https://github.com/google/zx/blob/main/docs/markdown.md). I want to make the prose primary and the commands incidental to executing the prose.

Even with that we'd get easier refactoring like extracting all the arg setup and diagnostics to a support function.

Reply:
> this script isn't a good candidate for that [=lights out].

got it.

> I tend to run them interactively by copying lines over to my shell.

I hope to avoid that. Very much not my style.

> If we want a CI test integrating agops I think that's worth writing fresh.

This script does represent a product scenario that's worth lots of testing. The extent to which any automated test shares code with the copy-to-my-shell script, I don't have strong feelings.

Reply:
> I hope to avoid that. Very much not my style.

Sounds worth a conversion to find a mutually agreeable design compatible with our workflows.